### ISSUE TITLE
Timeout config triggers errors.

### ISSUE THREAD

**Issue by EgorBu** [src-d/style-analyzer#365]
Hi, 
I tried to use https://github.com/src-d/lookout/blob/master/docs/configuration.md#advanced-fine-tuning to configure timeouts and failed. 
When I create `config.yml` with 
```
timeout:
  analyzer_review: 15m
  analyzer_push: 90m
  github_request: 30s
  git_fetch: 10m
  bblfsh_parse: 60s
```
it gives me `analyzer run: error: Unexpected line 0 in /io/config.yml: timeout:`.

**Comment by zurk** [comment]
Solved with new lookout-sdk https://github.com/src-d/lookout/issues/376

**Comment by zurk** [comment]
We should dump dependency on lookout-sdk-ml.

**Comment by vmarkovtsev** [comment]
Fixed in lookout-sdk-ml==0.4.0
